apps/webClient/src/views/Home/index.tsx

```diff
@@ -33,7 +33,7 @@ const Home: React.FC<Props> = ({}) => {
       <Box mt={12}>
         <Grid container>
           <Grid item container sm={3} justify="flex-start">
-            <NavButton route="ourwork" text="our work" />
+            <NavButton route="our-work" text="our work" vertical={true} />
           </Grid>
           <Grid item container sm={6} justify="center">
             <Container maxWidth="xs">
```

Comment:
this max width is still a little too big compared to the redesign. I can always give it a class, but this is as small as container goes without overriding the `xs` maxWidth I believe.

Reply:
I'm not sure I understand the point of this container.

The immediate child of a `Grid container` should be a `Grid item`

Reply:
With container 
![Screenshot from 2020-03-09 14-54-56](https://user-images.githubusercontent.com/16766024/76247538-176c5680-6216-11ea-8c30-0688eb3a6960.png)
No container 
![Screenshot from 2020-03-09 14-55-09](https://user-images.githubusercontent.com/16766024/76247559-1cc9a100-6216-11ea-84ba-4f367e326b6f.png)

Just a small difference in width of the text width. It's quite possible aswell that I am misusing grid. You can have a Grid be an item but also a container. I'm using this to center some things.

Reply:
Ahhh, got it.

I would:

1) remove container
2) increase `Grid spacing`
3) make the center grid item more narrow than `sm={6}`

Reply:
Also, using a max width here isn't the worst thing

Reply:
Appreciate you trying to go the MUI route, but I think (feel free to prove me wrong) that container maxWidth is generally to be used as a page level width reducer.  Grid item sizing is used to reduce width variably for columns

Reply:
Happy to sync more on this if I'm not being clear.

Reply:
UPDATE: No longer using a container here. Changed grid size to better container the content, pic below.